### PR TITLE
Updated Flask example

### PR DESCRIPTION
**Pull request by drageryd**
Data is no longer base64 encoded, and some format changes

### CODE REVIEW

File: python-flask/python-flask-example.py [hunk 1]
```diff
@@ -13,20 +13,18 @@ def log_payload(path):
...
+    data = request.json
+    if not isinstance(data, dict):
         return 'Payload is not JSON', 500
 
     # Get message data
-    timestamp = datetime.datetime.now()
-    message = payload.get('message',{})
-    if message:
+    device_id = data.get('deviceId', {})
+    if device_id:
```
Comment (pola) [inline python-flask/python-flask-example.py]: Could this be a `try` instead? We should really always be sending a `deviceId` anyway.

File: python-flask/python-flask-example.py [hunk 2]
```diff
@@ -13,20 +13,18 @@ def log_payload(path):
...
         # Get device ID
-        id = message.get('attributes',{}).get('deviceId','')
-        # Get base64 encoded data payload
-        data = base64.b64decode(message.get('data',''))
-        print('----- Got message from {} ({}) -------'.format(id, timestamp))
-        pprint.pprint(json.loads(data))
+        payload = data.get('payload', {})
+        timestamp = data.get('timestamp', datetime.datetime.now())
```
Comment (pola) [inline python-flask/python-flask-example.py]:
```suggestion
        timestamp = data.get('timestamp', '(no timestamp provided)')
```
A timestamp is always provided, but this makes it more clear in the very unlikely case that it isn't.

Comment (drageryd) [reply python-flask/python-flask-example.py]: true, changed it

File: python-flask/python-flask-example.py (hunk 1 above)
Comment (drageryd) [reply python-flask/python-flask-example.py]: Sure, updated original commit

**Review by pola — APPROVED** [review]

**Pull request merged**